src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -453,6 +460,9 @@ private bool TryGetWorkspaceFixersPriorityMap(Document document, [NotNullWhen(tr
 
                         if (!applicableDiagnostics.IsEmpty)
                         {
+                            // Add the CodeFix Provider Name to the parent CodeAction's CustomTags.
+                            action.AddCustomTag(fixerMetadata?.Name ?? fixer.GetTypeDisplayName());
```

Comment:
Do we want to add in case the fixerMetadata.Name is null? Seems like we wouldn't...

Reply:
Talked offline.this is for external cases

Reply:
I'll add a comment.